modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -986,7 +981,7 @@ impl Wallet {
             "New consensus height, syncing up",
         );
 
-        for height in (old_height + 1)..=(new_height) {
```

Comment:
Bleh. :D

Reply:
I think initially this code was operating on "block count" like `bitcoind`s `getblockcount` rpc, which is actually not a block count, but the height of the highest block (so inclusive range), which is probably the dumbest part of `bitcoind`, right before reversting txids.

Then I went on mission to change everything to use real `block_count`, but this code might have got transformed mechanically to match previous behavior. Wouldn't surprise me if I did this it. :D